examples/bzlmod/cross_compile/LLVM_Troubleshooting.md

```diff
@@ -0,0 +1,29 @@
+### LLVM Troubleshooting
+
+On older linux distributions (Ubuntu 16.04) you may encounter an error that C++ versions before C++ 14 are no longer
+supported. In this case, just install gcc version 7 or newer. This is rare corner case, but there are gcc backports for
+older distributions, so please upgrade your compiler if you ever see this error.
+
+On Ubuntu 20.04 you may see an error that a shared library called libtinfo.so.5 is missing. In that case, just install
+libtinfo via apt-get since its in the official 20.04 repo. To so, open a terminal and type:
+
+`
+apt update && apt install -y libtinfo5
+`
+
+The libtinfo5 library may have different package names on other distributions, but it is a well known
+issue. [See this SO discussion](https://stackoverflow.com/questions/48674104/clang-error-while-loading-shared-libraries-libtinfo-so-5-cannot-open-shared-o)
+for various solutions.
+
+On MacOX, it is sufficient to have the Apple Clang compiler installed.
```

Comment:
@keith @brentleyjones do y'all have specific recommendations or docs we can link to here around when to have Xcode CLT vs full install?